### PR TITLE
feat: Team page

### PR DESCRIPTION
**Pull request by miedziak**
* implement page based on design
* add reusable collapse component with animation

Resolves: OCM-15

### CODE REVIEW

File: src/components/Collapse/Collapse.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 2.4rem;
+`;
+
+const AnimatedContentWrapper = styled(animated.div)`
+  overflow: hidden;
+`
+
+// eslint-disable-next-line complexity
```
Comment (LWronka) [inline src/components/Collapse/Collapse.js]: May be move some if statements above component to improve readability and decrease complexity.

File: src/components/Collapse/ToggleIcon.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    enter: { opacity: 1, transform: 'scale(1)', position: 'relative' },
+    leave: { opacity: 0, transform: 'scale(0)',  position: 'absolute' },
+  });
+
+  return (
+    <IconContainer>
+      {transitions.map(({ item, key, props }) =>
+        (item
+          ? <AnimatedIcon key={key} style={props}><Icon icon={ChevronRightIcon} width='14px' height='14px' /></AnimatedIcon>
+          : <AnimatedIcon key={key} style={props}><Icon icon={CrossIcon} width='14px' height='14px' /></AnimatedIcon>)
```
Comment (LWronka) [inline src/components/Collapse/ToggleIcon.js]: `width` and `height` is needed here, svgs should have declared `viewbox` ?

File: src/components/pages/team/HeaderSection.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import Text from '../../Text';
+import Heading from '../../Heading';
+import puzzleGraphics from '../../../assets/puzzle-graphics.svg';
+import HeaderGridSection from '../HeaderGridSection';
+
+
```
Comment (LWronka) [inline src/components/pages/team/HeaderSection.js]: empty line

File: src/components/pages/team/HeaderSection.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      <br />
+      Ockam Builds For Builders.
+    </Heading>
+    <Text mt={4}>
+      Builders of connected device systems are going to change the world. Ockam is building the infrastructure and the open source tools that will enable any developer, of any skill level, to build these ecosystems to be secure, trustworthy, and interoperable.
+    </Text>
+  </HeaderGridSection>
+);
+
+HeaderSection.propTypes = {};
```
Comment (LWronka) [inline src/components/pages/team/HeaderSection.js]: empty `propTypes`

File: src/components/pages/team/JoinTeam.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import PageSection from "../PageSection";
+import Collapse from "../../Collapse/Collapse";
+import Text from "../../Text";
+import Button from "../../Button";
+import Link from "../../Link";
+import Heading from "../../Heading";
+
+const JoinTeam = () => {
+  return (
+    <PageSection>
```
Comment (LWronka) [inline src/components/pages/team/JoinTeam.js]: I think more readable will be move texts to another file or above component into array with objects, and component should be then more clear.

File: src/components/pages/team/ValuesSection.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import PageSection from '../PageSection';
+import { media } from '../../../utils/emotion';
```
Comment (LWronka) [inline src/components/pages/team/ValuesSection.js]: What do you think about add aliases to paths ?

File: src/components/pages/team/ValuesSection.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+`;
+
+const DescriptionBox = styled('div')`
+  grid-area: description;
+`;
+
+const QuatationBox = styled('div')`
+  grid-area: quotation;
+  border: 2px solid ${props => props.theme.colors.accentBackground};
+  border-radius: ${props => props.theme.radii.default};
```
Comment (LWronka) [inline src/components/pages/team/ValuesSection.js]: probably typo in `radii`

File: src/components/pages/team/ValuesSection.js [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  padding: 5rem 3rem 3rem 3rem;
+`;
+
+
```
Comment (LWronka) [inline src/components/pages/team/ValuesSection.js]: empty line

File: src/components/Collapse/ToggleIcon.js (hunk 2 above)
Comment (miedziak) [reply src/components/Collapse/ToggleIcon.js]: @LWronka Yeap, we could skip it normal use case, but it has to be that way to make this icon animation, not bouncing.

File: src/components/pages/team/JoinTeam.js (hunk 5 above)
Comment (miedziak) [reply src/components/pages/team/JoinTeam.js]: Good point, but I will skip it for now, cause this data will come thru `lever` API

File: src/components/pages/team/ValuesSection.js (hunk 6 above)
Comment (miedziak) [reply src/components/pages/team/ValuesSection.js]: @LWronka Good point, I would make it in separate PR, all at once.

File: src/components/pages/team/ValuesSection.js (hunk 7 above)
Comment (miedziak) [reply src/components/pages/team/ValuesSection.js]: @LWronka This one is correct, it comes from styled-system specification -> https://styled-system.com/theme-specification#key-reference

**Pull request merged**